src/extractors/custom/medium.com/index.js

```diff
@@ -1,22 +1,16 @@
 export const MediumExtractor = {
   domain: 'medium.com',
 
-  supportedDomains: ['trackchanges.postlight.com'],
-
   title: {
-    selectors: ['h1'],
+    selectors: [['meta[name="og:title"]', 'value'], 'h1'],
```

Comment:
I should have mentioned this, but:

If you can get title from the actual page rather than the meta tags, that's preferable, since often the meta title will be slightly different from the page title, and with Mercury, we aim first to select what the reader would see on the page.

Reply:
Makes sense, thanks! 💯